multi/stm32l4-multi/tty.c

```diff
@@ -429,26 +605,65 @@ int tty_init(void)
 		callbacks.set_cflag = tty_setCflag;
 		callbacks.signal_txready = tty_signalTxReady;
 
-		if (libtty_init(&ctx->tty_common, &callbacks, 512, baudrate) < 0)
+		if (libtty_init(&ctx->tty_common, &callbacks, 512, baudrate) < 0) {
```

Comment:
Perhaps libtty buffer can now be smaller? 512 is huge and was picked as a (somewhat not working) workaround for losing data over rx